include/lmms_constants.h

```diff
@@ -49,9 +49,13 @@ const float F_PI_SQR = (float) LD_PI_SQR;
 const float F_E = (float) LD_E;
 const float F_E_R = (float) LD_E_R;
 
+// Microtuner
+const unsigned int MaxScaleCount = 10;  //!< number of scales per project
+const unsigned int MaxKeymapCount = 10; //!< number of keyboard mappings per project
+
 // Frequency ranges (in Hz).
 // Arbitrary low limit for logarithmic frequency scale; >1 Hz.
-const int LOWEST_LOG_FREQ = 10;
+const int LOWEST_LOG_FREQ = 5;
```

Comment:
The spectrum analyzer will be able to display slightly lower frequencies on the full range log scale.

This becomes more relevant with the microtuner, since a custom tuning could more easily reach such low frequencies (why would the user do it is another question..). But even without it, the 10 Hz limit turns out to be a bit too high, since the lowest MIDI note has a frequency around 8.18 Hz in the default tuning. So this expands the range to include it as well.